### PR TITLE
add link to edit posts on github

### LINKED ISSUE #1504: Add one-click link to edit blog post page on GitHub
**Is your feature request related to a problem? Please describe.**
Some commands needed correction in Deno v1.4 release post. The viewer ended up cutting issues on deno core repo, and they were fixed in a PR.

Issues: https://github.com/denoland/deno/issues/7448, https://github.com/denoland/deno/issues/7449
PR: https://github.com/denoland/deno_website2/pull/1503

**Describe the solution you'd like**
Provide a one-click link to edit blog post page on GitHub repo, so that users can create PRs directly on deno_website2

**Describe alternatives you've considered**
[without knowledge blog post being open source] Using other means of communication to inform maintainers about the required corrections - like Discord, Twitter, Reddit etc.

**Additional context**
The AWS Developer Guides have such option in the footer.

Example: https://docs.aws.amazon.com/sdk-for-javascript/v2/developer-guide/welcome.html

<details>
<summary>Screenshot</summary>

Screenshot from 2020-09-13 10-24-50

</details>

### PR DESCRIPTION
**Pull request by tokiedokie**
fixes #1504

### CODE REVIEW

**Comment by lucacasonato** [comment]
@tokiedokie Thanks - the icon is not aligned correctly though:
image

File: pages/posts/[post].tsx [hunk 1]
```diff
@@ -95,6 +103,24 @@ const NewsPostPage = (props: Props) => {
...
+          href={`${postPath}${path}.md`}
+          className={`text-gray-500 hover:text-gray-900 transition duration-150 ease-in-out float-right mt-2`}
+        >
+          <span className="sr-only">GitHub</span>
+          <svg
+            className="h-6 w-6 inline"
+            fill="currentColor"
+            viewBox="0 0 24 24"
+          >
+            <title>Github | Deno</title>
```
Comment (lucacasonato) [inline pages/posts/[post].tsx:116]:
```suggestion
            <title>Edit on GitHub</title>
```

**Review by lucacasonato — APPROVED** [review]
Thanks, just one nit pick

File: pages/posts/[post].tsx (hunk 1 above)
Comment (tokiedokie) [reply pages/posts/[post].tsx]: I also update github icon title on manual

Comment (lucacasonato) [reply pages/posts/[post].tsx]: Perfect. Thanks :-)

**Pull request merged**